hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/meta/CkpMetadata.java

```diff
@@ -92,13 +92,14 @@ public void close() {
   // -------------------------------------------------------------------------
 
   /**
-   * Initialize the message bus, would clean all the messages
+   * Initialize the message bus, would keep all the messages.
    *
    * <p>This expects to be called by the driver.
    */
   public void bootstrap() throws IOException {
-    fs.delete(path, true);
-    fs.mkdirs(path);
+    if (!fs.exists(path)) {
```

Comment:
If a checkpoint succeed and the job crashes suddenly, meanwhile the JM restarts on another machine instance, the ckp metadata isn't keeped. This change only solves the scenario where JM is on the same machine. WDYT?